### PR TITLE
Accept the `docker_command` param being received from the apps.json endpoint

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@bruskiza & @Johanndutoit ready for review.
This builds on the `docker_command` app parameter that's being provided by the latest work on the goddard-hub-server.

File: scripts/setup.sh [hunk 1]
```diff
@@ -112,7 +112,16 @@ if [ ! -f /var/goddard/setup.lock ]; then
...
+				docker_command=`cat /var/goddard/apps.json | jq -r '.[] | select(.key == "$tkey") | .docker_command'`
+				if [ $docker_command = 'null' ]
+					then
+						docker_command_str="docker run --restart=always -p $tport:8080 -d $tkey"
+					else
+						# We evaluate the command against the current context
+						docker_command_str=${docker_command}
+					fi
+
+				cd /var/goddard/apps/$tkey && bash -c $docker_command
```
Comment (Johanndutoit) [inline scripts/setup.sh]: docker_command_str doesn't seem to be used ?

**Comment by Johanndutoit** [comment]
Would probably be good to provision the node in your office and just run this, but apart from variable not being used looks good.

**Comment by bruskiza** [comment]
Is this now ready for review again? @smn @Johanndutoit ?

**Comment by smn** [comment]
@bruskiza yeah it is ready for review

**Comment by bruskiza** [comment]
:+1: from me.

**Comment by smn** [comment]
Testing this on the node in our CT office.

**Comment by smn** [comment]
works on the CT node.

**Pull request merged**